modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -1033,7 +1033,8 @@ else if (interceptorVal != val0)
                     false,
                     updateCntr0,
                     null,
-                    topVer);
+                    topVer,
+                    false);
```

Comment:
Need to make default method without an explicit last parameter.
Because only one place in code base needs to specify this parameter.